### PR TITLE
Improve static type checking

### PR DESCRIPTION
**Pull request by kibablu**
Add type annotation to truncate Function on apnstruncate.py file
as part of the #201 issue

### CODE REVIEW

File: sygnal/apnstruncate.py [hunk 1]
```diff
@@ -38,7 +38,7 @@ def is_too_long(payload: dict, max_length: int = 2048) -> bool:
     return len(json_encode(payload)) > max_length
 
 
-def truncate(payload, max_length=2048):
+def truncate(payload: Dict[Any, Any], max_length: int = 2048) -> Dict[str, Any]:
```
Comment (callahad) [inline sygnal/apnstruncate.py:41]:
```suggestion
def truncate(payload: Dict[str, Any], max_length: int = 2048) -> Dict[str, Any]:
```

If we're returning a `Dict[str, Any]` we should probably restrict the inputs to be the same

**Review by callahad — CHANGES_REQUESTED** [review]
Looks good aside from one small tweak requested

File: sygnal/apnstruncate.py (hunk 1 above)
Comment (kibablu) [reply sygnal/apnstruncate.py]: Thank you :smiley:  I have made the changes

**Comment by callahad** [comment]
🎉  Thank you so much!

**Review by callahad — APPROVED** [review]

**Pull request merged**